app/src/main/java/com/lightricks/feedexercise/network/DataTransferObjects.kt

```diff
@@ -1,5 +1,22 @@
 package com.lightricks.feedexercise.network
 
-/**
- * todo: add Data Transfer Object data class(es) here
- */
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class FeedData(@Json(name = "templatesMetadata") val metadata: List<Metadata> = listOf()) {
+    @JsonClass(generateAdapter = true)
+    data class Metadata(
+        @Json(name = "configuration") val configuration: String = "",
+        @Json(name = "id") val id: String = "",
```

Comment:
I am a bit religious about separating server side API from structures. I have seen horrors on a C# based servers (they use capital camel case: Id, Configuration). And if you decide to refactor id (with Shift+F6), for example, it will break the code.